searchcore/src/apps/proton/proton.cpp

```diff
@@ -228,6 +238,7 @@ App::Main()
                 EV_STOPPING("servicelayer", "clean shutdown");
             }
             protonUP.reset();
+            transport.ShutDown(true);
```

Comment:
consider shutting down transport before deleting proton, to reduce chance of core dumps